1.2-Introduction-to-Musig.ipynb

```diff
@@ -153,31 +154,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Generate nonces\n",
-    "k0 =  # TODO: implement\n",
-    "k1 =  # TODO: implement\n",
-    "k2 =  # TODO: implement\n",
-    "R0 =  # TODO: implement\n",
+    "# Generate nonces and nonce points\n",
+    "# Method: generate_schnorr_nonce() with no argument generates a random nonce, or it can be provided\n",
+    "# a seed to generate a nonce deterministically.\n",
```

Comment:
There shouldn't be any deterministic nonces in musig

Reply:
You're right. I'm setting the nonce manually here for testing purposes. I've updated the comment. Let me know what you think.

Reply:
idk, why not just use a random nonce?

Reply:
Changed this to set the nonce explicitly with a comment explaining why